src/main/java/edu/oakland/production/display02/DisplayCommInterfaceImplementation.java

```diff
@@ -0,0 +1,40 @@
+package edu.oakland.production.display02;
+
+import edu.oakland.helper.admin.LocationDataPoint;
+import edu.oakland.helper.admin.TrackData;
+import edu.oakland.production.display02.DisplayComm2WayInterface;
+import edu.oakland.production.display02.DisplayCommInterface;
+import java.lang.IllegalArgumentException;
+
+/**
+* This class receives the rfid, evaluates it,
+* then passes it along to Comm2Way.
+*
+*/
+public class DisplayCommInterfaceImplementation implements DisplayCommInterface {
+
+  private DisplayComm2WayInterface comm2Way;
+
+  /**
+  * Creates a DisplayCommInterfaceImplementation to receive the rfid.
+  * param DisplatComm2WayInterface: The class that DisplayCommInterface will pass the rfid to.
+  *
+  */
+  public DisplayCommInterfaceImplementation(DisplayComm2WayInterface displayComm2WayInterface) {
+    if (displayComm2WayInterface == null) {
+      throw new IllegalArgumentException("processor cannot be null");
```

Comment:
This should be as follows, as it's not really the processor:
```suggestion
      throw new IllegalArgumentException("displayComm2WayInterface cannot be null");
```